src/config.h

```diff
@@ -172,6 +172,16 @@
 #define VALKEY_NO_SANITIZE(sanitizer)
 #endif
 
+#if defined(__SANITIZE_ADDRESS__)
+/* GCC */
+#define VALKEY_ADDRESS_SANITIZER 1
+#elif defined(__has_feature)
+#if __has_feature(address_sanitizer)
+/* Clang */
+#define VALKEY_ADDRESS_SANITIZER 1
+#endif
+#endif
```

Comment:
Another option is to use the GCC macro for this. I might be cleaner.

```suggestion
#elif defined(__has_feature) && !defined(__SANITIZE_ADDRESS__)
#if __has_feature(address_sanitizer)
/* Clang */
#define __SANITIZE_ADDRESS__
#endif
#endif
```

Reply:
I actually prefer the original code, since if someone wants to check how `VALKEY_ADDRESS_SANITIZER` is defined, it will learn that there are differences between GCC and Clang.